<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>StudReg</class>
 <widget class="QWidget" name="StudReg">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>640</width>
    <height>480</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <item row="0" column="0">
    <widget class="QLabel" name="label_2">
     <property name="text">
      <string/>
     </property>
     <property name="pixmap">
      <pixmap resource="style.qrc">:/rc/gradientLogo.png</pixmap>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item row="0" column="2">
    <widget class="QLabel" name="label_3">
     <property name="text">
      <string/>
     </property>
     <property name="pixmap">
      <pixmap resource="style.qrc">:/rc/gradientLogo.png</pixmap>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <layout class="QGridLayout" name="gridLayout">
     <item row="0" column="0">
      <widget class="QFrame" name="studframe">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>200</width>
         <height>350</height>
        </size>
       </property>
       <property name="autoFillBackground">
        <bool>false</bool>
       </property>
       <property name="frameShape">
        <enum>QFrame::Box</enum>
       </property>
       <widget class="QLabel" name="userLabel">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>30</y>
          <width>131</width>
          <height>21</height>
         </rect>
        </property>
        <property name="text">
         <string>Choose Username</string>
        </property>
       </widget>
       <widget class="QLabel" name="passLabel">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>90</y>
          <width>131</width>
          <height>21</height>
         </rect>
        </property>
        <property name="text">
         <string>Create Password</string>
        </property>
       </widget>
       <widget class="QLabel" name="classLabel">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>150</y>
          <width>151</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>Class Code (optional)</string>
        </property>
       </widget>
       <widget class="QLineEdit" name="userEntry">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>50</y>
          <width>171</width>
          <height>31</height>
         </rect>
        </property>
       </widget>
       <widget class="QLineEdit" name="passEntry">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>110</y>
          <width>171</width>
          <height>31</height>
         </rect>
        </property>
       </widget>
       <widget class="QLineEdit" name="classEntry">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>170</y>
          <width>171</width>
          <height>31</height>
         </rect>
        </property>
       </widget>
       <widget class="QPushButton" name="regButton">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>270</y>
          <width>171</width>
          <height>32</height>
         </rect>
        </property>
        <property name="text">
         <string>Register</string>
        </property>
       </widget>
       <widget class="QPushButton" name="cancelButton">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>310</y>
          <width>171</width>
          <height>32</height>
         </rect>
        </property>
        <property name="text">
         <string>Cancel</string>
        </property>
       </widget>
       <widget class="QLabel" name="classLabel_2">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>210</y>
          <width>151</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>Date of Birth:</string>
        </property>
       </widget>
       <widget class="QDateEdit" name="dobEntry">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>230</y>
          <width>171</width>
          <height>31</height>
         </rect>
        </property>
        <property name="displayFormat">
         <string>yyyy-MM-dd</string>
        </property>
        <property name="date">
         <date>
          <year>2016</year>
          <month>12</month>
          <day>31</day>
         </date>
        </property>
       </widget>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="style.qrc"/>
 </resources>
 <connections/>
</ui>
